.spelling

```diff
@@ -115,6 +116,7 @@ filesystem
 firefox-data-docs
 FirefoxReality
 FireTV
+FLOAT64
```

Comment:
Any reason you didn't use backticks for the types? Seems like an odd thing to add to the dictionary.

Reply:
"int64 was already in the dictionary," which I agree is not the strongest argument

Reply:
Backticked the types.